### ISSUE TITLE
Can't use zio-test with scala-cli

### ISSUE THREAD

**Issue by bbarker** [VirtusLab/scala-cli#426]
I created a[ scala-cli gist](https://gist.github.com/bbarker/ccfc54cc7afce875703183fb9d6d7d2c/49424a63b698328cf51df0a78fb054bcdbfd4fda), the Main file runs fine, but the test, though it compiles, does not run. I noticed an munit example worked fine when I tried it, so I'm guessing scala-cli either needs to support zio-test for this to work, or alternatively, there is some problem with zio(-test).

**Comment by alexarchambault** [comment]
@bbarker Do you know why zio test is split between `zio-test` and `zio-test-sbt`? Most other Scala test frameworks just embed their sbt test framework implementation in their main module. Plus I guess it wouldn't only be useful from sbt, but also from Mill or Bloop (and now Scala CLI too).

**Comment by lwronski** [comment]
If I replace `dev.zio::zio-test` to `dev.zio::zio-test-sbt` it started working with scala-cli:
```
$ cat ExampleSpec.scala
// using scala 3.1.0
// using lib dev.zio::zio::2.0.0-M6-2
// using lib dev.zio::zio-test-sbt::2.0.0-M6-2

import zio.*
import zio.test.*
import zio.test.Assertion.equalTo

object ExampleSpec extends DefaultRunnableSpec {
  def spec = suite("ExampleSpec")(
    testM("integer addition is associative") {
      val left = 2
      UIO(assert(left)(equalTo[Int](2)))
    }
  )
}
$ scala-cli test  ExampleSpec.scala
Compiling project (Scala 3.1.0, JVM)
Compiled project (Scala 3.1.0, JVM)
+ ExampleSpec
  + integer addition is associative
Ran 1 test in 183 ms: 1 succeeded, 0 ignored, 0 failed
Done
```


In the next version of ScalaCLI, we will add a more user-friendly message, if scala-cli didn't find test framework implementation in main module.

**Comment by lwronski** [comment]
Fixed in #505, 

@bbarker Thanks for reporting, in the next version of scala-cli users should see an error message:
`zio-test found in the class path, zio-test-sbt should be added to run zio tests with Scala CLI."`
if try to run `zio` test without `zio-test-sbt` on classPath.

**Comment by bbarker** [comment]
Amazing, that's some nice dev UX :-)
